__tests__/app.test.js

```diff
@@ -185,4 +185,18 @@ describe("GET /api/articles/:articleid/comments", () => {
         .then( ( { body: { msg } } ) => {
             expect(msg).toBe("Invalid article_id")
         })})
+})
+
+
+describe("GET /api/users", () => {
+    
+    test("200: responds with an array of users", () => {
+        return request(app)
+        .get("/api/users/")
+        .expect(200)
+        .then( ( { body: { users} } ) => {
+            expect(users).toEqual(data.userData)
```

Comment:
Tiny thing but I'd be careful with requiring in the raw data for comparison, you should make your own assertions about what the data should look like according to the db structure, as the raw data won't always match. Hardly the end of the world though :)